tools/sgcloudendpoints/deprecated.go

```diff
@@ -0,0 +1,100 @@
+package sgcloudendpoints
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/tools/sgdocker"
+	"go.einride.tech/sage/tools/sggcloud"
+)
+
+// ConfigID returns the ID of the latest config for the provided Cloud Endpoints service.
+// Deprecated: Use sgcloudendpoints.DeployConfig instead.
```

Comment:
`DeployConfig` requires a config to be passed. I don't see a function to substitute being able to get what the latest config is, is that not needed?

Reply:
Instead of querying for the latest config - we can use the return value from `DeployConfig.

Reply:
Yes but that requires deploying something while the user might just want to inquire the latest config but without looking deeper into current usages I do get a feeling that the old API is likely being used immediately after deploying anyway and for this use-case this is a better API

Reply:
Yes! I'm pushing a POC PR after we merge this one - we use it immediately after deploying to build the container image